docs/src/templates/pages/documentation/css/index.hbs

```diff
@@ -1,141 +1,59 @@
-
-When using these mixins, the easiest way to write your media queries is to specify the breakpoint name; the @include-media library will then convert this into a media query with a value in ems to provide a more consistent experience for users that use text zoom.
-
-
-#### Media query example
-
-```scss
-/**
- * Using @include-media you can specify breakpoints
- * using the map key's defined in Fozzie's [breakpoint map](https://github.com/justeat/fozzie/blob/8a1094ae5a467e9e2afd86a4aab62f9ea65b1bee/src/scss/settings/_variables.scss#L65-L72)
- */
-@include media('>=mid') {
-	a {
-		color: darkgoldenrod;
-	}
-}
-/* Is the same as: */
-@media (min-width: 48em) {
-	a {
-		color: darkgoldenrod;
-	}
-}
-```
-
-#### Better media query example
-Rather than having all of your media queries for different widths stored in separate `SCSS` files or placed at the bottom of each `SCSS` partial, it is suggested that you make use of Sass' inline media queries.
-
-This means that all styles related to an element are grouped together, for example:
+<a name="preprocessors"></a><h2 class="sg-sectionHeading">Preprocessors</h2>
 
-```scss
-a {
-	padding: 1em;
+We use a combination of [Sass](https://sass-lang.com/) and [PostCSS](https://postcss.org/) to author our stylesheets.
 
-	@include media('>=mid') {
-		padding: 2em;
-	}
-}
-```
+For more information on how we use these tools, check out our documentation sections on [Sass](/css/sass) and [PostCSS](/css/postcss).
 
 
-<a name="modernizr"></a>
 
-### Modernizr
+<a name="progressive"></a><h2 class="sg-sectionHeading">Progressive Enhancement</h2>
 
-At JUST EAT, we strongly advocate feature detection over browser or UI sniffing.
+We strongly believe in using progressive enhancement so that we can provide a service to as many users as we can.  Those using modern browsers will get the best possible experience, with those on [older browsers](/general/browser-support) still able to view and use our site.
 
-Our minimal framework includes a custom build of Modernizr. To update this with an extra feature test that isn’t already included in our build, ensure you copy the url in the current [modernizr.min.js](https://github.je-labs.com/I18N-Production-ROTW/ConsumerWeb/blob/master/src/ConsumerWeb/Assets/src/js/libs/modernizr.min.js) so that the current tests are preserved in the new build.
+As part of this effort, we strongly recommend using feature detection over browser or UI sniffing.
 
-For full information on using Modernizr, see [their documentation](http://modernizr.com) or see our [Javascript documentation]({{ baseUrl }}/js/#js-modernizr) to find out where it is included.
+Where needed, we use Modernizr to help us do this, and where possible [CSS feature queries via `@supports`](https://caniuse.com/#feat=css-featurequeries) (browser support allowing – ironically).
```

Comment:
Is this still true for the global microsites?

Reply:
I've left this in for now – we don't atm but until the GlobalWeb pages are built out I didn't know if this would be irrelevant or not, so thought it best to keep it in!